internal/core/command/init.go

```diff
@@ -50,40 +40,7 @@ func NewBootstrap(router *mux.Router) *Bootstrap {
 
 // BootstrapHandler fulfills the BootstrapHandler contract and performs initialization needed by the command service.
 func (b *Bootstrap) BootstrapHandler(ctx context.Context, wg *sync.WaitGroup, _ startup.Timer, dic *di.Container) bool {
-	loadRestRoutes(b.router, dic)
 	v2.LoadRestRoutes(b.router, dic)
 
-	// TODO: there is an outstanding known issue (https://github.com/edgexfoundry/edgex-go/issues/2462)
-	// 		that could be seemingly be solved by moving from JIT initialization of these external clients to static
-	// 		init on startup, like registryClient and configuration are initialized.
-	// 		Doing so would cover over the symptoms of the bug, but the root problem of server processing taking longer
-	// 		than the configured client time out would still be present.
-	// 		Until that problem is addressed by larger architectural changes, if you are experiencing a bug similar to
-	//		https://github.com/edgexfoundry/edgex-go/issues/2421, the correct fix is to bump up the client timeout.
-	configuration := container.ConfigurationFrom(dic.Get)
-	lc := bootstrapContainer.LoggingClientFrom(dic.Get)
-
-	// initialize clients required by the service
-	dic.Update(di.ServiceConstructorMap{
-		container.MetadataDeviceClientName: func(get di.Get) interface{} {
-			return metadata.NewDeviceClient(local.New(configuration.Clients[clients.CoreMetaDataServiceKey].Url() + clients.ApiDeviceRoute))
-		},
-		errorContainer.ErrorHandlerName: func(get di.Get) interface{} {
-			return errorconcept.NewErrorHandler(lc)
-		},
-		V2Container.MetadataDeviceClientName: func(get di.Get) interface{} { // add v2 API MetadataDeviceClient
-			return V2Clients.NewDeviceClient(configuration.Clients[clients.CoreMetaDataServiceKey].Url() + V2Routes.ApiDeviceRoute)
-		},
-		V2Container.MetadataDeviceProfileClientName: func(get di.Get) interface{} { // add v2 API MetadataDeviceProfileClient
-			return V2Clients.NewDeviceProfileClient(configuration.Clients[clients.CoreMetaDataServiceKey].Url() + V2Routes.ApiDeviceProfileRoute)
-		},
-		V2Container.MetadataDeviceServiceClientName: func(get di.Get) interface{} { // add v2 API MetadataDeviceServiceClient
-			return V2Clients.NewDeviceServiceClient(configuration.Clients[clients.CoreMetaDataServiceKey].Url() + V2Routes.ApiDeviceServiceRoute)
-		},
-		V2Container.DeviceServiceCommandClientName: func(get di.Get) interface{} { // add v2 API DeviceServiceCommandClient
-			return V2Clients.NewDeviceServiceCommandClient()
-		},
```

Comment:
V2 Clients are using in the core-command's application layer

Reply:
Humm, not sure why those obvious V2 lines got removed. Will revert.

Reply:
restored